android/app/src/main/java/app/organicmaps/car/SurfaceRenderer.java

```diff
@@ -220,7 +220,7 @@ public void enable()
     mMap.onStart();
     mMap.setCallbackUnsupported(this::reportUnsupported);
     mMap.setMapRenderingListener(this);
-    mMap.updateMyPositionRoutingOffset(0);
+    UiThread.runLater(() -> mMap.updateMyPositionRoutingOffset(0));
```

Comment:
Why this one only?
I suppose that ```mMap.updateMyPositionRoutingOffset``` call is semantically the same as ```mMap.setMapRenderingListener```, no?

Reply:
All this code is run in the UI thread anyway but I want to schedule this function call for a later not overcross with the same call from the device which may break the offset for AA. But I covered the device's call with `if (isDeviceDisplayUsed)`. So, it shouldn't be a problem but to be 100% sure

Reply:
Please add an explaining comment into the code, it looks unclear.